### ISSUE TITLE
Bug - Truncated - content cannot be focused via keyboard

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#7270]
**Describe the problem**
This is an issue with 
- [ ] Patternfly 5  
- [x] Patternfly 6  
- [ ] other

Please give a clear and concise description of the problem. Which components are affected?

From the 2024 Q4 a11y audit.

In the Truncate component, the truncated content cannot be focused via keyboard which would be necessary to trigger a tooltip to display the entire content (note that a tooltip doesn't need to be implemented in the Core examples)

**How do you reproduce the problem?**
Provide steps to reproduce. A [codepen](http://codepen.io) or [codesandbox](https://codesandbox.io/) demonstrating the problem is appreciated.

Go to the [Truncate examples](https://www.patternfly.org/components/truncate/html) and try tabbing through the page. Notice how the truncated content never receives focus.

**Expected behavior**
A clear and concise description of the expected behavior.

The truncated content would receive focus.

**Is this issue blocking you?**
List the workaround if there is one.

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**

**What is your product and what release date are you targeting?**

**Any other information?**
